### DISCUSSION TITLE
missed monero transactions

### DISCUSSION THREAD

**Discussion by hardwardbern** [btcpayserver/btcpayserver#6526] [Technical Support]
Hello team, I encountered a strange situation. 

I had monero accepted via btcpay set up on my site. Everything worked correctly before, but recently I discovered that monero transactions are not displayed in my monero wallet gui. I don't understand where the monero transactions that were made via btcpay. I checked the settings and see that there are no errors. How can I understand where the funds go?

I also tried the Feathner wallet via a secret phrase and after all
![Uploading Снимок экрана 2024-12-28 в 17.40.59.png…]()
 the synchronization the transactions were not displayed

**Comment by pavlenex** [comment]
Perhaps @sethforprivacy or other monero users can help you.
